setup.py

```diff
@@ -0,0 +1,16 @@
+from setuptools import setup
+
+setup(
+    name="legion",
+    version="0.1.0",
```

Comment:
go ahead and change the version to 1.0.0

Reply:
done